### PR TITLE
[SIG-2541] fixes styling in 24 hour map

### PR DESCRIPTION
**Pull request by aditudorache**
This PR:
- Fixes the styling in the 24 hour map

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentOverviewPage/components/Map/components/DetailPanel/index.js [hunk 1]
```diff
@@ -18,11 +18,17 @@ const Panel = styled.div`
   justify-content: space-between;
 `;
 
+const IncidentLink = styled(AscLink)`
+  font-size: 16px;
+  color: ${themeColor('primary')};
+  font-weight: 700;
```
Comment (janjaap) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/Map/components/DetailPanel/index.js:24]: This style rule doesn't seem to be applied:
Screenshot 2020-04-20 at 09 15 45

Comment (aditudorache) [reply src/signals/incident-management/containers/IncidentOverviewPage/components/Map/components/DetailPanel/index.js]: Strange, couldn't reproduce it on my development environment. I have checked with the designer and agreed to use a standard asc link here.

File: src/signals/incident-management/containers/IncidentOverviewPage/components/SubNav/index.js [hunk 2]
```diff
@@ -15,6 +15,8 @@ export const MapHeading = styled(Heading).attrs({
 })`
   margin: 0;
   font-size: 16px;
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+  font-weight: 400;
```
Comment (janjaap) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/SubNav/index.js:19]: These changes are not necessary; font is already `Demi`

Comment (aditudorache) [reply src/signals/incident-management/containers/IncidentOverviewPage/components/SubNav/index.js:19]: Unfortunately it is, it inherits `font-weight: 700`  from `Heading`

Comment (jpoppe) [reply src/signals/incident-management/containers/IncidentOverviewPage/components/SubNav/index.js:19]: I can confirm this (the inheritance).

**Pull request merged**